packages/frontend/src/store/timeEntries.ts

```diff
@@ -221,3 +251,123 @@ function updateTimeEntries(state: State, paramEntries: FrontendTimentrie[]) {
   }
   state.timeEntries = newTimeEntries;
 }
+
+function getRelevantYear() {
+  const today = new Date();
+  return today.getFullYear();
+}
+
+function getRelevantMonths(): Date[] {
+  const today = new Date();
+  const lastMonth = new Date(today.getFullYear(), today.getMonth() - 1);
+  const twoMonthsAgo = new Date(
+    lastMonth.getFullYear(),
+    lastMonth.getMonth() - 1
+  );
+  const relevantMonths = [twoMonthsAgo, lastMonth, today];
+  return relevantMonths;
+}
+
+function filterTimeEntriesByRelevantMonths(
+  allTimeEntriesAllTasks: TimeEntriesDateFormated[][]
+) {
+  const relevantMonths = getRelevantMonths();
+  function isInRelevantMonths(date: Date): boolean {
```

Comment:
Relevent er relativt til hva?
```suggestion
  function isInTheLastThreeMonths(date: Date): boolean {
```